static/js/search.js

```diff
@@ -56,7 +78,18 @@ export function setup_refine_search() {
     for (const el of refine_cced) {
         $(el).click(() => {
             const cced = el.id.split(":")[1];
-            $('#cced').val(cced);
+            const old_cced = $('#cced').val();
```

Comment:
@Carol217 - this inner stuff can be factored out - I know you're time limited right now, so I'll go ahead and do it to show an example.